src/Traits/FormattingTrait.php

```diff
@@ -229,7 +229,7 @@ public function toUnixString(): string
      * Returns the quarter
      *
      * @param bool $range Range.
-     * @return int|array 1, 2, 3, or 4 quarter of year or array if $range true
+     * @return float|array 1, 2, 3, or 4 quarter of year or array if $range true
      */
     public function toQuarter(bool $range = false)
     {
```

Comment:
Just notice that the `ceil` function will always return `float` type, and it should use `float|array` type hints instead.

Reply:
What I was going to say is that "int" is expected and "float" is wrongly returned :)
I think this is a bug because both from expectation and code using this, the float value makes no sense whatsoever.

Reply:
Thanks for your reply, @dereuromark.

To resolve this issue, I think we should cast the `ceil` function to be `int` and revert returned type hints to be `int|array`.

What do you think about this :)?

Reply:
@dereuromark, the above approach I have been updated on the latest PR commit, please feel free to review :).